lib/src/runner/browser/content_shell.dart

```diff
@@ -59,9 +60,16 @@ class ContentShell extends Browser {
         // make sure it's not expired and that the remote debugging port worked.
         // Any errors from this will always come before the "Running without
         // renderer sandbox" message.
-        while (await stderr.moveNext() &&
+        while (await stderr
+                .moveNext()
+                .timeout(_errorTimeout)
+                .catchError((_) => false) &&
             !stderr.current.endsWith("Running without renderer sandbox") &&
-            !stderr.current.contains("Running without the SUID sandbox")) {
+            !stderr.current.contains("Running without the SUID sandbox") &&
+            // Error messages on Mac can get gobbled, we must assume that it
+            // it started up successfully.
```

Comment:
It doesn't produce any meaningful output other than the 'kqueue' message above. As far as I can tell, content-shell 50 reliably does this on Mac only. However, other versions may also have this issue. When it does happen, unfortunately we can't tell if the bind was successful.

I've reduced the timeout. Looking forward to deprecating content-shell.